### DISCUSSION TITLE
How can I find the accuracy of a Neural Network model ?

### DISCUSSION THREAD

**Discussion by sonymag** [kyleskom/NBA-Machine-Learning-Sports-Betting#384] [Q&A]
@kyleskom Hello friend!
Can u tell me how can I understand what accuracy have my  Neural Network model?
Thx in advance!
![Screenshot_1](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/83603430/0fb68bc6-ad8b-47b8-a322-35faa7f14165)

**Comment by kyleskom** [comment]
Look into [tensorboard](https://www.tensorflow.org/tensorboard)
